pom.xml

```diff
@@ -108,6 +108,11 @@
       <artifactId>commons-compress</artifactId>
       <version>1.18</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value-annotations</artifactId>
+      <version>1.6.2</version>
```

Comment:
I still think we should not do this, but if we go ahead anyway; this dependency absolutely must have scope compile.

Reply:
compile is the default scope?

Reply:
You're right. I was thinking of compileOnly in gradle. Maven doesn't really have that, but we need it here. Without it, I feel even more strongly that we should not introduce this dependency.

Reply:
We can make it `provided` (which is almost the equivalent of `compileOnly`). So I talked to eamonn about this, and he said there shouldn't be a problem doing that since the annotations are processed at compile time but they will be dangling in the compile classes.

Reply:
"dangling"?  With `RetentionPolicy.SOURCE` they should never be mentioned in the resulting .class files.

```
@Retention(RetentionPolicy.SOURCE)
@Target(ElementType.TYPE)
public @interface AutoValue {
```

Reply:
I believe that has changed to `CLASS`:

https://github.com/google/auto/blob/1339e4038c23b8d667316ac4f179a8dac35685e4/value/src/main/java/com/google/auto/value/AutoValue.java#L44